starlette/requests.py

```diff
@@ -121,11 +121,18 @@ def url_for(self, name: str, **path_params: typing.Any) -> str:
     raise RuntimeError("Receive channel has not been made available")
 
 
+async def empty_send(message: Message) -> None:
+    raise RuntimeError("Send channel has not been made available")
```

Comment:
@jlaine I'm not sure it should error - a failed server push only affects performance.

Reply:
I mimicked the `request.body()` test. What we are catching here is a programming error internal to `starlette` isn't it?